training-net/src/Controllers/MoviesController.cs

```diff
@@ -54,9 +54,12 @@ public IActionResult Create()
         [HttpPost("Create")]
         public IActionResult Create(MovieViewModel mvm)
         {
-            var movie = new Movie { ID = mvm.ID, Title = mvm.Title, ReleaseDate = mvm.ReleaseDate, Genre = mvm.Genre, Price = mvm.Price };
+            if(ModelState.IsValid)
+            {
+            var movie = new Movie { ID = mvm.ID ?? default(int), Title = mvm.Title, ReleaseDate = mvm.ReleaseDate, Genre = mvm.Genre, Price = mvm.Price };
```

Comment:
again... why `default(int)`. it seems to be wrong. when we create a Movie in this case, we don't set an ID, because its autogenerated when we add and save it.